airflow/www/utils.py

```diff
@@ -41,6 +43,47 @@
 from airflow.www.widgets import AirflowDateTimePickerWidget
 
 
+def datetime_to_string(value: Optional[DateTime]) -> Optional[str]:
```

Comment:
I think this can be just (the stdlib) `datetime.datetime` and not restricted to `pendulum`. Also there's a similar function in `views.py` so we should refactor that to use this function as well.